test-clients/src/main/java/com/hedera/services/bdd/suites/misc/CannotDeleteSystemEntitiesSuite.java

```diff
@@ -0,0 +1,173 @@
+package com.hedera.services.bdd.suites.misc;
+
+/*-
```

Comment:
Will this test be added to nightly regression?

Reply:
Right now, it's added to Continuous Integration since these are still simple, low TPS functional tests. I am not sure if we will add this test suites to other regression test workflow. If there is one, the ``AWS-Daily-Services-Comp-Basic-4N-1C``` shall be the good destination for this test suite. Yeah, I guess it will be good to add this test in that regression test for now. I will do that for now. 

(I had a discussion with Nosh yesterday, his suggestion was to put high TPS (like several thousand TPS) tests in the regression tests to fully utilize those AWS instances' power. While the tests with basic functional tests, like ```AWS-Daily-Services-Comp-Basic-4N-1C``` should be moved back to simple CircleCi workflow, like CI workflow to reduce the cost.  I will create a new issue for this effort.)

Reply:
Understood. Thanks !